package/CHANGELOG

```diff
@@ -21,6 +21,7 @@ The rules for this file:
   * 2.0.0
 
 Fixes
+  * ITPParser now accepts relative paths
```

Comment:
Can you include the issue number here please?

Can you also add yourself to the author list above and the `AUTHORS` file?

Reply:
Oh, I think I am already on the authors list on both places..

Reply:
Sorry about that, just the issue number then.